StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/VerticalSavedPaymentOptionsViewControllerSnapshotTests.swift

```diff
@@ -37,4 +37,12 @@ final class VerticalSavedPaymentOptionsViewControllerSnapshotTests: STPSnapshotT
         sut.view.autosizeHeight(width: 375)
         STPSnapshotVerifyView(sut.view)
     }
+
+    private func generatePaymentMethods() -> [STPPaymentMethod] {
+        return [STPFixtures.paymentMethod(),
+                STPFixtures.usBankAccountPaymentMethod(),
+                STPFixtures.usBankAccountPaymentMethod(bankName: "BANK OF AMERICA"),
+                STPFixtures.usBankAccountPaymentMethod(bankName: "STRIPE"),
+                STPFixtures.sepaDebitPaymentMethod(), ]
```

Comment:
We have two ways to provide fixtures unfortunately.  THe old Obj-C way is static methods on `STPFixtures` class. THe new way is to provide extensions on the thing. See `STPFixtures+PaymentSheet.swift`. It already has `STPPaymentMethod._testSEPA` and I think it's a more ergonomic and scalable way we should use going forward.

Reply:
Should probably make a ticket to migrate us off of `STPFixtures`

Reply:
Filed [here!](https://jira.corp.stripe.com/browse/RUN_MOBILESDK-3166). I'll leave as-is for now but I'll pick this up next time I'm on run.